examples/FadeExample.m

```diff
@@ -59,6 +59,9 @@ - (void)viewDidLoad {
 @end
```

Comment:
Add an explicit conformity of `TransitionWithCustomDuration` to FadeTransition so that the compiler enforces the method signature.